packages/core/src/utils/helper.ts

```diff
@@ -160,4 +161,35 @@ export function omit<T extends object, U extends keyof T>(
   );
 }
 
+export function isPluginIncluded(
+  config: LibConfig,
+  pluginName: string,
+): boolean {
+  return Boolean(
+    config.plugins?.some((plugin) => {
+      if (typeof plugin === 'object' && plugin !== null && 'name' in plugin) {
+        return plugin.name === pluginName;
+      }
+      return false;
+    }),
+  );
+}
+
+export function checkMFPlugin(config: LibConfig): boolean {
+  if (config.format !== 'mf') {
+    return true;
+  }
+
+  const added = isPluginIncluded(config, 'rsbuild:module-federation-enhanced');
+  if (!added) {
+    logger.warn(
+      `${color.green('mf format')} is enabled, but the ${color.blue(
+        '@module-federation/rsbuild-plugin',
+      )} plugin was not found in plugins.`,
```

Comment:
```suggestion
      `${color.green('"mf"')} should be used with ${color.blue(
        '@module-federation/rsbuild-plugin',
      )}," consider installing and adding to plugins. Check the documentation (a link here) to get start with "mf" output.`,
```

Reply:
The rough suggestion in not the final code, you need to tweak it. Error messages should:

1. tell user where they're wrong
2. what they need to do to solve the problem
3. some extra links for complex configuration

Reply:
done